src/app/cases/styles.ts

```diff
@@ -65,7 +65,7 @@ export const CardColumn = styled.div`
 
 export const CaseDetailsContainer = styled.div`
```

Comment:
for the `ListingCard` column, I believe @jinkang-0  added styling for a custom scrollbar. Do we wanna add it here as well?? @kyrenetam curious what you think of the look of this as well! Here's a screenshot (this ensures the scrollbars are always consistent regardless of the device):

<img width="1440" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/a07485ea-90fd-4795-aaea-ad4d75c5b7ee">

Reply:
ooo, during worksesh i asked @jinkang-0 if he could add it, but i'll coordinate this further

Reply:
from worksesh: i add scroll